packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.jsx

```diff
@@ -24,9 +24,8 @@ import {
 } from '../../../constants';
 import { useFolder } from '../../../hooks/useFolder';
 import { usePersistentState } from '../../../hooks/usePersistentState';
-import { getBreadcrumbDataCM, toSingularTypes } from '../../../utils';
-import getAllowedFiles from '../../../utils/getAllowedFiles';
-import getTrad from '../../../utils/getTrad';
+import { getBreadcrumbDataCM, toSingularTypes, getTrad } from '../../../utils';
+import { getAllowedFiles } from '../../../utils/getAllowedFiles';
```

Comment:
why are some files imported from utils index and others from individual files?

Reply:
I changed the file, now I export every utils from the index